12-offer-encoding.md

```diff
@@ -58,6 +58,19 @@ The merchant-pays-user flow (e.g. ATM or refund):
 3. The merchant confirms the *invoice_node_id* to ensure it's about to pay the correct
    person, and makes a payment to the invoice.
 
+The pay-mobile-user flow (e.g. paying a friend back to their mobile node):
+1. The mobile user supplies some always-online node with a static (i.e. 
+   `payment_hash`-less) invoice to return on its behalf. This always-online node may 
+   be the mobile user's channel counterparty, wallet vendor, or another node on the 
+   network that it has an out-of-band relationship with.
+2. The mobile user publishes an offer that contains blinded paths that terminate
+   at the always-online node.
```

Comment:
The blinded paths, that isn't a strict requirement, or is it?

Also wondering how pathfinding works for the sender. They only get to do one hold htlc without retries, so it must be first time right?